### ISSUE TITLE
Error in pick_up no voice mode.

### ISSUE THREAD

**Issue by jumper-d-1981** [emxsys/callattendant#163]
I am having no ability to block.  Restarted several times, took everything apart and set everything back up from scratch.  It just rings through.  It will not block or pick up.  Says it cannot go into voice mode.  Seen someone suggest updating the firmware and tried the newest drivers on the us robotics site but I cannot figure out how to run the update to the driver/firmware.  I need an assist.
![2022-02-04 16_45_39-Window](https://user-images.githubusercontent.com/80989663/152607509-16c57113-2627-4189-a885-5a4d71452c6f.png)

**Comment by jumper-d-1981** [comment]
Some research on this matter...
I looked into the AT commands that enable voice.  Seems that it is supposed to be AT+FCLASS=8 for voice mode which is mentioned in the code...but I installed this modem onto my Windows machine...and the driver...and tried to AT that command and I get an ERROR as well.  To verify that it was taking commands I used +FCLASS=1 which will return OK but no other number.  Checked the manufacturers webpage, and found that it is indeed supposed to have that option...leaving me to wonder what is going on.

I really need to figure this out.  What I don't understand is how it worked initially but suddenly does not seem to.

I also tried a seperate PI and the only thing I used was this modem, I used new SD and everything...installed and same outcome...I am wondering if it is a driver issue...but I cannot figure out how to get this driver to install...

Thanks.
